### PR TITLE
Material UI v5 migration

### PR DESCRIPTION
**Pull request by sBouzols**

### CODE REVIEW

**Comment by sBouzols** [comment]
waiting to use commons-ui version with MUI V5 to merge this PR
That's the reason why Github CI don't pass. The current commons-ui used here (for now) needs @material-ui/* packages which were removed.

File: src/components/dialogs/filters-editor.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const useStyles = makeStyles(() => ({
 
 const CustomNativeSelect = withStyles((theme) => ({
     select: {
-        color: theme.palette.type === 'light' ? '#000 !important' : '#fff',
+        color: theme.palette.mode === 'light' ? '#000 !important' : '#fff',
```
Comment (laurgarn) [inline src/components/dialogs/filters-editor.js:30]: Has this been corrected through the upgrade, by any chance ?

File: src/components/dialogs/filters.js [hunk 2]
```diff
@@ -54,7 +53,7 @@ export const CountriesSelection = ({ initialValue, onChange, disabled }) => {
             onChange={(oldVal, newVal) => onChange(newVal)}
             options={Object.keys(countriesList.object())}
             getOptionLabel={(code) => countriesList.get(code)}
-            renderInput={(props) => <TextField {...props} variant="outlined" />}
+            renderInput={(props) => <TextField {...props} />}
```
Comment (laurgarn) [inline src/components/dialogs/filters.js:56]: Could this be a return to the default ?

**Review by laurgarn — APPROVED** [review]

File: src/components/dialogs/filters.js (hunk 2 above)
Comment (sBouzols) [reply src/components/dialogs/filters.js:56]: this is the default value for `TextField` `variant` prop then it's useless to specify it in V5

File: src/components/dialogs/filters-editor.js (hunk 1 above)
Comment (sBouzols) [reply src/components/dialogs/filters-editor.js:30]: I don't even know why we use a CustomNativeSelect here ? 
not in the scope I would say

**Review by jonenst — APPROVED** [review]

File: src/components/app.test.js [hunk 3]
```diff
@@ -39,12 +44,16 @@ it('renders', async () => {
...
                     <Provider store={store}>
-                        <ThemeProvider theme={createTheme({})}>
-                            <SnackbarProvider hideIconVariant={false}>
-                                <CssBaseline />
-                                <App />
-                            </SnackbarProvider>
-                        </ThemeProvider>
+                        <StyledEngineProvider injectFirst>
+                            <ThemeProvider
+                                theme={createTheme(adaptV4Theme({}))}
```
Comment (niconoir) [inline src/components/app.test.js]: Can't `adaptV4Theme` be removed here too ?

**Review by niconoir — COMMENTED** [review]
Still a call to adaptV4Theme which can be removed.

File: src/components/app.test.js (hunk 3 above)
Comment (sBouzols) [reply src/components/app.test.js]: removed

**Pull request merged**